src/filter/blur.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+import Image from '../image';
+import convolution from '../operator/convolution';
+
+// first release of mean filter
+export default function meanFilter(k, boundary){
+
+    this.checkProcessable({
```

Comment:
checkProcessable requires as first parameter the name of the procedure. In this case 'meanFilter'